mani_skill/envs/tasks/digital_twins/utils/real_ppo_agent_eval.py

```diff
@@ -0,0 +1,114 @@
+import os
+from dataclasses import dataclass
+from typing import Optional
+
+import gymnasium as gym
+import torch
+import tyro
+
+from examples.baselines.ppo.ppo_rgb import Agent
```

Comment:
this works? there may be import issues given people might not clone the examples folder, only the ppo_rgb.py file

Reply:
I don't have a good solution actually. Unless we can store the Agent class itself in the model save file? Would be easier that way.